.travis/staging.sh

```diff
@@ -0,0 +1,16 @@
+output=$(lektor project-info --output-path)
+cp -r $output staging
+cd staging
+
+# remove files over 100mb because github can´t handle them (only happens on one video)
+find -size +100M | xargs rm
+
+# push to staging repository
+git init
+git add .
+git commit -m "staging commit"
+git remote add origin https://$LEKTOR_STAGING_KEY@github.com/maxbachmann-staging/website-staging$(($TRAVIS_PULL_REQUEST % 5)).git
+git push --force origin master
```

Comment:
![force](https://www.commitstrip.com/wp-content/uploads/2019/03/Strip-Le-cot%C3%A9-obscur-de-la-force-650-finalenglish.jpg "Be carefully!")

*[Source](http://www.commitstrip.com/en/2019/03/11/the-force/)*